docker-compose.yml

```diff
@@ -14,9 +29,11 @@ services:
       - ./migrations/dlu:/docker-entrypoint-initdb.d
     networks:
       - darkflame
+    ports:
+      - 3306:3306 # (optional) ports only exposed so that DB management tools can connect
```

Comment:
I would remove this. This is just a security risk and you don't need to expose the port to be able to connect to the database via an external tool. If you are on the host you can connect with the container ip and on a remote server you can use a ssh tunnel.

Reply:
Done